js/models.js

```diff
@@ -0,0 +1,32 @@
+App.Models.Todo = Backbone.Model.extend({
+    defaults: {
+        'isComplete': false,
```

Comment:
One thing I like to do is add some comments for what the data props will be for this model. Helps other developers know what kind of data they can expect to set and get. 
It's not an expectation, but it's always nice to see. It's also just a good habit to get into.